### PR TITLE
Release Notes updates for K8ssandra 1.3.0

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by johnsmartco** [comment]
Converted this PR to Draft - wait for K8ssandra 1.3.0 release.

**Comment by johnsmartco** [comment]
Ready for review now, but will wait for ok from @jdonenine and @bradfordcp w.r.t. 1.3.0 release before merging to main / publish.

File: docs/content/en/release-notes/_index.md [hunk 1]
```diff
@@ -35,25 +38,25 @@ Also deployed is Stargate, an open source data gateway that lets you interact pr
...
+  * 3.11.10
   * 3.11.9
-  * 3.11.10 (default)
-  * 4.0 rc1
+  * 3.11.8
+  * 3.11.7
 * DataStax Kubernetes Operator for Apache Cassandra ([cass-operator](https://github.com/k8ssandra/cass-operator)) 1.7.1
-* Management API for Apache Cassandra ([MAAC](https://github.com/datastax/management-api-for-apache-cassandra)) 0.1.25
-* [Stargate](https://github.com/stargate/stargate) 1.0.18
+* Management API for Apache Cassandra ([MAAC](https://github.com/datastax/management-api-for-apache-cassandra)) 0.1.26
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated - thanks Jeff.

File: docs/content/en/release-notes/_index.md [hunk 2]
```diff
@@ -74,8 +77,55 @@ To manually restart Stargate nodes:
...
+* Upgrade to reaper-operator 0.3.3 and Reaper 2.3.0.
+* Upgrade from Stargate 1.0.18 to 1.0.29.
+* Upgrade from Medusa 0.10.1 to 0.11.0.
+* Upgrade from Reaper 2.2.2 to 2.2.5.
+* Integrate Fossa component/license scanning, [#812](https://github.com/k8ssandra/k8ssandra/issues/812).
+* Upgrade medusa-operator to 0.3.3, [#905](https://github.com/k8ssandra/k8ssandra/issues/905).
+
+### New features
+
+* Upgrade from Management API 0.1.25 to 0.1.26 to provide support for Cassandra 4.0.0-RC2, [#890](https://github.com/k8ssandra/k8ssandra/issues/890).
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated - thanks Jeff.

**Review by jdonenine — APPROVED** [review]
Looks good, thanks!

**Pull request merged**